wiki/Ranking/en.md

```diff
@@ -0,0 +1,39 @@
+---
+stub: true
+tags:
+  - pp
+  - leaderboards
+  - ranking
+---
+
+# Ranking
+
+**Ranking** (alternatively called "leaderboards") is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
+
+All but the individual beatmap rankings can be navigated to by hovering the `rankings` menu on the website header. For the individual beatmap ranking, they can be found on their beatmap info page.
+
+## Performance points ranking
+
+*Main page: [Performance points](/wiki/Performance_points)*
+
+The performance points ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by the total number of performance points an individual player has, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).
+
+## Score ranking
+
+The score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.
+
+## kudosu! ranking
```

Comment:
```suggestion
## Kudosu ranking
```

Reply:
@Joehuu suggested to use `kudosu!` (stylized like that) in https://github.com/ppy/osu-wiki/pull/3867#discussion_r527128471, I'm also leaning that way too.... I was about to add it as a term in the ASC to be spelt like that.